hummingbot/core/data_type/trade.py

```diff
@@ -20,7 +20,7 @@ class Trade(namedtuple("_Trade", "trading_pair, side, price, amount, order_type,
     order_type: OrderType
     market: str
     timestamp: float
-    trade_fee: TradeFee
+    trade_fee: AddedToCostTradeFee
```

Comment:
Should the type be a union of all TradeFee classes?

Reply:
Fixed for `TradeFeeBase` [here](https://github.com/CoinAlpha/hummingbot/pull/21/commits/5400d92aef10a3b6541bfc1d1b5b2a10ed54a7e2)